README.md

```diff
@@ -20,10 +20,10 @@ automatically installed by the plugin.
 
 # How to use
 
-The plugin JAR needs to be defined in your build script. It is directly available on Maven Central. Alternatively, you can download it from GitHub and deploy it to your local repository. The following code snippet shows an example on how to retrieve it from Maven Central:
+The plugin needs to be defined in your `build.gradle`. It is directly available on Maven Central. Alternatively, you can download it from GitHub and deploy it to your local repository. The following code snippet shows an example on how to retrieve it from Maven Central:
 
 ### Using plugins block
-Since the appengine-gradle-plugin is not published to the gradle plugin portal, you must refrence it from the Central Maven repository.  Add the maven central resolution to your `settings.gradle`
+The `appengine-gradle-plugin` is not published to the gradle plugin portal, you must refrence it from the Central Maven repository.  Add the maven central resolution to your `settings.gradle`
```

Comment:
I think "since ..." is correct unless you terminate the statement by a period like "... to the gradle plugin portal."? And "since" sounds natural to me.

Reply:
Yeah I was supposed to add a `so` after the the comma, but I'll just revert to `Since`